daemon/algod/api/server/v2/account.go

```diff
@@ -360,11 +385,12 @@ func AppParamsToApplication(creator string, appIdx basics.AppIndex, appParams *b
 
 // AssetParamsToAsset converts basics.AssetParams to generated.Asset
 func AssetParamsToAsset(creator string, idx basics.AssetIndex, params *basics.AssetParams) generated.Asset {
+	frozen := params.DefaultFrozen
 	assetParams := generated.AssetParams{
 		Creator:       creator,
 		Total:         params.Total,
 		Decimals:      uint64(params.Decimals),
-		DefaultFrozen: &params.DefaultFrozen,
+		DefaultFrozen: &frozen,
 		MetadataHash:  byteOrNil(params.MetadataHash[:]),
```

Comment:
The MetadataHash field also need to be handled. As is, it would not create a copy, but rather use the original pointer. ( since it's a true array and not a slice )

Reply:
yep, done